honeybee_energy/material/glazing.py

```diff
@@ -102,7 +103,7 @@ def __init__(self, identifier, thickness=0.003, solar_transmittance=0.85,
         self._solar_reflectance_back = None
         self._visible_reflectance = 0
         self._visible_reflectance_back = None
-
+        self._user_data = None
```

Comment:
This is unnecessary but I'll fix it after we merge

Reply:
ARG how did I miss this I just looked through everything!